docs/pbc/all/dynamic-multistore/202404.0/base-shop/import-stores.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Add stores by importing data
+description: This document shows how to import a store and related entities.
+past_updated: Sep 25, 2024
+template: howto-guide-template
+last_updated: Nov 17, 2023
+---
+
+This document describes how to import a store with related entities.
```

Comment:
This is not DMS specific dataimport.
Question: do we have somewhere integration guide for spryker-core feature? If so - please move it there.

Reply:
Dataimport enables store only for DMS. For non-DMS mode configuration should be updated too.
I can rename an article: enable a store for storefront. Or smh like this